src/Files/FileLoader.php

```diff
@@ -124,6 +125,10 @@ private function tryLoadFile($fileName, $extension)
 				{
 					return new LangFile($this->debug, $fileName, $this->rundir);
 				}
+				if (trim(strtolower($dir[0])) == 'migrations')
```

Comment:
You should check. Pretty sure @Derky had the dev team make that change. I know the change was made, but not sure if it was kept.